### PR TITLE
fix #27086 amazon.com.tr

### LINKED ISSUE #27086: amazon.com.tr
### Issue URL (Ads)
[https://www.amazon.com.tr/ref=nav_logo](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.amazon.com.tr%2Fref%3Dnav_logo)
### Comment
Username: @PentagramTr
> Potential counter/tracker scripts.
> 
> `https://www.amazon.com.tr/gp/uedata?at&v=0.202580.0&id=W3M3669SG0BWFEAD136A&ctb=1&m=1&sc=W3M3669SG0BWFEAD136A&pc=14122&at=14122&t=1545029610984&ec=2&ecf=0&pty=Gateway&spty=desktop&pti=desktop&tid=N8DARK5KYNWSZ5RRHHAX&aftb=1`
> 
> `https://www.amazon.com.tr/ah/ajax/counter?ctr=desktop_ajax_atf&exp=1545029713111&rId=W3M3669SG0BWFEAD136A&mkId=A33AVAJ2PDY3EV&h=b318abd483d45b584c75f42ce9c9c36290ec7589aa926c84814b1bc29b8d7d16`
> 
> `https://unagi-eu.amazon.com/1/events/com.amazon.csm.nexusclient.prod`
> 
> Pixel
> 
> `https://fls-eu.amazon.com/1/batch/1/OP/A33AVAJ2PDY3EV:259-6536170-8744732:4PRKT5Q0HJB68CXCFHN6$uedata=s:%2Fgp%2Fuedata%3Fat%26v%3D0.202580.0%26id%3D4PRKT5Q0HJB68CXCFHN6%26ctb%3D1%26m%3D1%26sc%3D4PRKT5Q0HJB68CXCFHN6%26pc%3D14094%26at%3D14094%26t%3D1545030455603%26ec%3D2%26ecf%3D0%26pty%3DGateway%26spty%3Ddesktop%26pti%3Ddesktop%26tid%3DN8DARK5KYNWSZ5RRHHAX%26aftb%3D1:14095`

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>
<details>
  <summary>Screenshot 3</summary>

  Screenshot 3
</details>
<details>
  <summary>Screenshot 4</summary>

  Screenshot 4
</details>

### System configuration

Information | value
--- | ---
Platform: | Ext
AdGuard version: | 2.10.14
Browser: | Chrome
Filters: | AdGuard Turkish,<br>AdGuard Annoyances,<br>AdGuard Base,<br>NoCoin Filter List,<br>AdGuard Spyware,<br>AdGuard Social Media

### PR DESCRIPTION
**Pull request by slavaleleka**
https://github.com/AdguardTeam/AdguardFilters/issues/27086

https://github.com/AdguardTeam/AdguardFilters/issues/27242

### CODE REVIEW

**Comment by PentagramTr** [comment]
@slavaleleka

Rule doesn't work on the mobile site.
`amazon.com.tr###desktop-banner-stripe`

**Comment by PentagramTr** [comment]
Now it is not working in the desktop version.

<details><summary>Screenshot:</summary>

www amazon com tr


</details>
<br />

@slavaleleka

File: SpywareFilter/sections/specific.txt [hunk 1]
```diff
@@ -69,6 +69,12 @@ tensaigakkou.ru###custom_html-16
 ||growingio.com^$third-party
 ||d.line-cdn.net/*/torimochi.js/*/torimochi.js
 ||d.line-scdn.net/*/torimochi.js/*/torimochi.js
+||amazon.com.tr/*/uedata?
```
Comment (Alex-302) [inline SpywareFilter/sections/specific.txt]: Better `/uedata?`. And remove old rules.

**Review by Alex-302 — CHANGES_REQUESTED** [review]
Better `/uedata?`(like in EasyPrivacy). And remove old rules.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**